### PR TITLE
Feature :: Improve forms diplay

### PR DESCRIPTION
**Pull request by vdestraitt**
This PR mainly consists in aesthetics:

- Clean duplicated CSS
- Improve styling: text sizing of labels, placeholders & buttons, text color, title alignment and  autocomplete/multiselect items highlight color
- Improve naming: labels & title

**Before:**

**After:**

**Before:**

**After:**

### CODE REVIEW

File: src/components/stepforms/DomainStepForm.vue [hunk 1]
```diff
@@ -40,7 +40,7 @@ export default class DomainStepForm extends BaseStepForm<DomainStep> {
 
   @State domains!: string[];
 
-  readonly title: string = 'Change your domain';
+  readonly title: string = 'Select a domain';
```
Comment (Dalanir) [inline src/components/stepforms/DomainStepForm.vue]: We open the DomainStep form only for the edition, I think for choosing it we've got a special CTA

Comment (vdestraitt) [reply src/components/stepforms/DomainStepForm.vue]: Actually "Change" sounds less natural English to my ears in such a context. To say "Change your domain" I would rather say "Select a new domain" but I am not an linguistic expert !!

Also, I may decide not to "change" the domain, I can leave it unchanged eventually. The 100% truth is in the selection, not the change :)

And my last answer will be: we are not searching for an important CTA like a payment button, so I think we can be quite flexible on synonyms ;)

**Comment by vdestraitt** [comment]
> One comment plus to store #a7a7a7 hex code in var and we're good, thanks for that :)

Done thanks !

**Comment by vdestraitt** [comment]
> Seems really good to me !
> Just one thing I'm not sure if the color of the placeholder text has enough contrast for accessibility :
> image
> 
> Update : I Check it on a web accessibility tool (source https://webaim.org/resources/contrastchecker/).
> Results : it's not ok
> image

**Comment by vdestraitt** [comment]
> Seems really good to me !
> Just one thing I'm not sure if the color of the placeholder text has enough contrast for accessibility 
> Update : I Check it on a web accessibility tool

@zegonz thanks I replaced this color with a much darker grey !

File: src/components/stepforms/DomainStepForm.vue (hunk 1 above)
Comment (Dalanir) [reply src/components/stepforms/DomainStepForm.vue]: 👍

**Pull request merged**